### PR TITLE
Upload users file 

### PR DESCRIPTION
**Pull request by mahdibenromdhane**

### CODE REVIEW

File: src/assets/i18n/fr.json [hunk 1]
```diff
@@ -149,6 +149,8 @@
     "status": "Statut",
     "revoke": "Révoquer",
     "edit": "Editer",
+    "import_instructions": "Pour importer les données, veuillez fournir une fichier .csv file contenant les Propriété(s): '{{properties}}'",
```
Comment (ClaudeROSSI) [inline src/assets/i18n/fr.json:152]: typo - une/un fichier

**Review by ClaudeROSSI — CHANGES_REQUESTED** [review]
just a typo in the french texts!

File: src/assets/i18n/fr.json (hunk 1 above)
Comment (mahdibenromdhane) [reply src/assets/i18n/fr.json]: fixed

**Review by ClaudeROSSI — APPROVED** [review]
approved

**Pull request merged**